lib/l10n/app_en.arb

```diff
@@ -951,5 +954,291 @@
+  "@runRepairWarning": {
+    "description": "Message shown when an error occurs during the downloads repair step of the Isar downloads migration, which requires a server connection."
+  },
+  "downloadSettings": "Download Settings",
+  "@downloadSettings": {},
+  "showPlaylistSongs": "Show Items with Unknown Library.",
+  "@showPlaylistSongs": {
+    "description": "Title of setting that shows songs/albums from an unknown library in all libraries when on, or in no libraries when off."
+  },
+  "showPlaylistSongsSubtitle": "Some items may be downloaded with an unknown library.  They can show in all libraries, or none.",
+  "@showPlaylistSongsSubtitle": {
+    "description": "Description of setting that shows songs/albums from an unknown library in all libraries when on, or in no libraries when off."
+  },
+  "maxConcurrentDownloads": "Max Concurrent Downloads",
+  "@maxConcurrentDownloads": {
+    "description": "Title of setting controlling max concurrent downloads."
+  },
+  "maxConcurrentDownloadsSubtitle": "Increasing concurrent downloads may allow increased downloading in the background but will delay the effect of some settings changes and may cause some downloads to fail if very large, or cause lag in some cases.",
+  "@maxConcurrentDownloadsSubtitle": {
+    "description": "Description of setting controlling max concurrent downloads."
+  },
+  "maxConcurrentDownloadsLabel": "{ count} Concurrent Downloads",
+  "@maxConcurrentDownloadsLabel": {
+    "description": "Label of values on max concurrent downloads slider",
+    "placeholders": {
+      "count": {
+        "type": "String"
+      }
+    }
+  },
+  "downloadsWorkersSetting": "Download Worker count",
+  "@downloadsWorkersSetting": {
+    "description": "Title of setting controlling download worker count."
+  },
+  "downloadsWorkersSettingSubtitle": "Amount of workers for syncing metadata and deleting downloads.  Increasing download workers may speed up download syncing and deleting, especially when server latency is high, but can introduce lag.",
+  "@downloadsWorkersSettingSubtitle": {
+    "description": "Description of setting controlling download worker count."
+  },
+  "downloadsWorkersSettingLabel": "{ count} Download Workers",
+  "@downloadsWorkersSettingLabel": {
+    "description": "Label of values on download worker count slider",
+    "placeholders": {
+      "count": {
+        "type": "String"
+      }
+    }
+  },
+  "syncOnStartupSwitch": "Automatically Sync Downloads at Startup",
+  "@syncOnStartupSwitch": {
+    "description": "Title of setting for automatically triggering a throttled resync on app startup."
+  },
+  "preferQuickSyncSwitch": "Prefer Quick Syncs",
+  "@preferQuickSyncSwitch": {
+    "description": "Title of setting that skips completed albums while syncing."
+  },
+  "preferQuickSyncSwitchSubtitle": "When performing syncs, fully downloaded songs and albums will be skipped.  Download repair will always perform a full sync.",
```

Comment:
"When performing syncs, some typically static items (like songs and albums) will not be synced. Download..."